fstracecheck.in

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 
+if [ $(uname -m) = i686 ]; then
```

Comment:
Note this. Without it, fstracecheck tries to do both -m32 and -m64 at once and fails

Reply:
No, this is a hack for what is ultimately a bug in fstracecheck. There are two options: switch to fstracecheck2 by integrating and enabling the compilation db plugin, or add a substitution for CCFLAGS and add -m32 or -m64 depending on wether -m32 is present in its value.